python-lib/text_extraction/__init__.py

```diff
@@ -0,0 +1,54 @@
+import docx
+from io import BytesIO
+import logging
+import os
+import pypandoc
+import pypdfium2 as pdfium
+import tempfile
+
+
+logger = logging.getLogger(__name__)
+
+
+def try_download_pandoc():
```

Comment:
[nitpick]
We could simplify to `download_pandoc` or `download_pandoc_binaries`, no need for the `try` to be in the function name